src/i18n/messages/en.messages.json

```
@@ -12,42 +12,33 @@
       "removeLabel": "Remove {suggestion} from history queries",
       "title": "History Queries"
     },
-    "querySuggestions": {
-      "title": "Suggestions"
-    },
     "nextQueries": {
       "title": "Next Queries"
     },
     "nextQueriesGroup": {
       "title": "You may be interested",
       "message": "This is what other shoppers searched after <span class=\"x-text x-text--bold\">\"{query}\"</span>"
     },
-    "identifierResults": {
-      "title": "SKU Search"
-    },
     "recommendations": {
       "title": "Top Clicked"
     },
     "facets": {
       "gender": "GENDER",
       "categoryPaths": "CATEGORIES",
-      "price": "PRICE",
-      "usage": "USAGE",
-      "season": "SEASON"
+      "price": "PRICE"
     },
     "filters": {
-      "all": "All"
+      "all": "Show more"
```

Shouldn't this still be 'All'?

```suggestion
      "all": "All"
```